applications/StructuralMechanicsApplication/custom_strategies/custom_strategies/mechanical_explicit_strategy.hpp

```diff
@@ -89,16 +93,13 @@ class MechanicalExplicitStrategy
         bool CalculateReactions = false,
         bool ReformDofSetAtEachStep = false,
         bool MoveMeshFlag = true)
-        : SolvingStrategy<TSparseSpace, TDenseSpace, TLinearSolver>(rModelPart, MoveMeshFlag) {
+        : SolvingStrategy<TSparseSpace, TDenseSpace, TLinearSolver>(rModelPart, MoveMeshFlag),
+          mpScheme(pScheme),
+          mReformDofSetAtEachStep(ReformDofSetAtEachStep),
```

Comment:
what are u doing with the ReformDof ?

please take into accoutn that using dofs is quite slow, so try to minimize their usage in an explicit context

to be more clear, they are not the dofs per se to be slow. it is the access to the dofs, so the call to

node->pGetdof

if you have a doflist already initialized you can well work with that

Reply:
The thing is that this member variable is not used, I just changed the way the constructor works

Reply:
If I remove the variable that would be a behavior change. For a future PR

Reply:
In conclussion: It was already there, is not used, I just move the way the constructor works